crates/bevy_gizmos/src/arrows.rs

```diff
@@ -125,7 +125,14 @@ impl<'w, 's, T: GizmoConfigGroup> Gizmos<'w, 's, T> {
         end: Vec2,
         color: impl Into<Color>,
     ) -> ArrowBuilder<'_, 'w, 's, T> {
-        self.arrow(start.extend(0.), end.extend(0.), color)
+        let length = (end - start).length();
+        ArrowBuilder {
+            gizmos: self,
+            start: start.extend(0.),
+            end: end.extend(0.),
+            color: color.into(),
+            tip_length: length / 10.,
+        }
```

Comment:
Why did you change this?

Reply:
I changed it to be consistent with the 3d arrow drawing method, but I can remove that.